platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/events/span/Span.kt

```diff
@@ -77,16 +84,24 @@ class Span internal constructor(
     fun end(
         result: SpanResult,
         fields: Map<String, String>? = null,
+        endTimeInMs: Long? = null,
     ) {
         logger?.apply {
+            val durationMs: Long =
+                if (endTimeInMs != null && startTimeInMs != null) {
+                    endTimeInMs - startTimeInMs
```

Comment:
so this means that for the single-shot api `logSpan()` both start and stop logs would appear back to back in the timeline even though technically the start should've happened at `startTimeInMs`, are we ok with that?

Reply:
also since they are passing `endTimeInMs` for the end this would mean that the timestamp won't be accurate either

Reply:
Sorry I didn't understand the first point? In the one-shot we don't send start

Reply:
Ah I guess I missed the `emitStartLog` part